writeups/quic-bit-grease.txt

```diff
@@ -0,0 +1,175 @@
+
+(1) What type of RFC is being requested (BCP, Proposed Standard, Internet
+Standard, Informational, Experimental, or Historic)? Why is this the proper type
+of RFC? Is this type of RFC indicated in the title page header?
+
+Proposed Standard. This is the proper type for a simple extension to the use of
+the QUIC wire image.
+
+(2) The IESG approval announcement includes a Document Announcement Write-Up.
+Please provide such a Document Announcement Write-Up. Recent examples can be
+found in the "Action" announcements for approved documents. The approval
+announcement contains the following sections:
+
+Technical Summary:
+
+The "wire image" of QUIC packets is described in QUIC invariants (RFC 8999).
+Little about this image is visible to passive observers. One field, the
+so-called QUIC bit, resides at the second-to-most significant bit of the first
+byte of every QUIC packet. RFC 9000 requires the QUIC bit to be set to a value
+of 1, with the purpose being to allow QUIC traffic to be easily distinguisable
+from other UDP-based traffic when they share a UDP port. This document defines a
+behavioural extension, negotiated using transport parameters, that permits
+endpoints to use any value the QUIC bit. The intention of this
+extension is to exercise variation in the wire image in order to support future
+QUIC changes before systems ossify.
+
+Working Group Summary:
+
+The document is suucint. During adoption and WGLC there was sufficient review
+and no major issues were raised or incidents occured.
+
+Document Quality:
+
+The document is succint and describes how to vary one bit, using
+well-established mechanisms. There are several implementations of the grease bit
+extension and several interoperable deployments deployed on the Internet.
+
+No special review has been required.
+
+Personnel:
+
+Lucas Pardue is the document shepherd. Zahed Sarker is the AD.
+
+(3) Briefly describe the review of this document that was performed by the
+Document Shepherd. If this version of the document is not ready for publication,
+please explain why the document is being forwarded to the IESG.
+
+I have reviewed this document thoroughly and personally implemented the
+extension, which took 4 hours including testing. The scope of the extension is
+focused, and is straightforward to implement for any person familiar with QUIC.
+
+(4) Does the document Shepherd have any concerns about the depth or breadth of
+the reviews that have been performed?
+
+No
+
+(5) Do portions of the document need review from a particular or from broader
+perspective, e.g., security, operational complexity, AAA, DNS, DHCP, XML, or
+internationalization? If so, describe the review that took place.
+
+No special review is required.
+
+(6) Describe any specific concerns or issues that the Document Shepherd has with
+this document that the Responsible Area Director and/or the IESG should be aware
+of? For example, perhaps he or she is uncomfortable with certain parts of the
```

Comment:
We need to update the shepherd writeup template

Reply:
I can't fix that here though

Reply:
I believe the IETF is working on an update to the template but until that is formally delivered, we're stuck with what we have.